.wordlist.txt

```diff
@@ -304,6 +304,7 @@ Rickle
 RoCE
 Roofline
 Ryzen
+rst
```

Comment:
RST (all cap) was added on line 297 of this file. Do we also want to add rst (lowercase)?